### DISCUSSION TITLE
VAX VMSやHP NonStop Serverへの対応について

### DISCUSSION THREAD

**Discussion by sayurin** [ffftp/ffftp#275] [General]
一応、VAX VMSやHP NonStop Serverへの対応が行われている。しかし、その後の拡張で、不十分な個所もありそうで、結果的にこれらの環境では動かなくなっているのではないか。
例えば、ディレクトリ区切りを追加する際は `/` から `.` へ切り替える処理が存在する。
https://github.com/ffftp/ffftp/blob/9f908c1dba09f78b9f0a7c103e68873c5f5d4df9/src/misc.c#L350-L363
ところが、ディレクトリ作成する際の解析では `/` しか見ていない。
https://github.com/ffftp/ffftp/blob/9f908c1dba09f78b9f0a7c103e68873c5f5d4df9/src/ftpproc.c#L1041

対応を維持するのであれば、仕様を確認しオーバーホールを行うべき。維持しないのであれば、コードを削除すべき。どちらがいいのだろうか。

**Comment by narareimei** [comment]
OpenVMS上で動くシステムの保守をしています。FFFTPを利用していますが、未だにVer1.99を使っています。出来ることなら最新バージョンを使っていきたいので、メンテしていただければ嬉しいです。

ディレクトリ解析以外では、ファイル一覧取得処理もVMSへの対応が不十分であることがわかっています。VMSではファイル名が16文字を超えると２行に分割されるので、1行に結合して解析する必要があります。
